dotCMS/src/main/java/com/dotmarketing/portlets/workflows/util/WorkflowImportExportUtil.java

```diff
@@ -204,6 +209,23 @@ public void importWorkflowExport(final WorkflowSchemeImportExportObject importer
 		}
 	}
 
+	private Role getAnonRole () {
+		return Try.of(()->APILocator.getRoleAPI().loadCMSAnonymousRole()).getOrNull();
+	}
+
+	private WorkflowAction validateAction(final WorkflowAction action) {
+
+		final String nextAssign = action.getNextAssign();
+		final Role role = Try.of(()->APILocator.getRoleAPI().loadRoleById(nextAssign)).getOrNull();
```

Comment:
why not:

```
final Role role = Try.of(()->APILocator.getRoleAPI().loadRoleById(nextAssign)).getOrElse(getAnonRole());
```

Reply:
I have tried this approach initially, but didn't work since the loadRoleById returns null if does not exists (instead of an exception) so the annon is being never reached